### PR TITLE
HTML: improve highlights & injections

### PR DESCRIPTION
**Pull request by stsewd**

### CODE REVIEW

File: queries/html/injections.scm [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ((style_element
   (raw_text) @css))
 
+((attribute
+   (attribute_name) @_attr
+   (quoted_attribute_value (attribute_value) @css))
+ (#match? @_attr "^style$"))
```
Comment (vigoux) [inline queries/html/injections.scm:7]: Why not using `eq` here?

Comment (stsewd) [reply queries/html/injections.scm]: changed

**Review by vigoux — APPROVED** [review]
Could yo urebase on latest master?

**Comment by theHamsta** [comment]
@vigoux  manual rebase might not be necessary in such situations. Our merge button is already set to rebase and when it's green then rebase should lead to no conflicts. I'm wondering whether this would not just do the se thing.

**Comment by vigoux** [comment]
Let's see

**Comment by vigoux** [comment]
Let's see

**Pull request merged**